torch/_inductor/codegen/cpp.py

```diff
@@ -1651,11 +1666,14 @@ def is_load_only_block(self, sub_graph: torch.fx.Graph):
                     break
 
                 # Create and record the context
-                opt_ctx = OptimizationContext()
+                if OptimizationContext.key in _node.meta:
+                    opt_ctx = _node.meta[OptimizationContext.key]
```

Comment:
Does that mean the `store` node could be executed multiple times?

Reply:
I think the `store` node may have `opt_ctx` in `legalize_bf16`. So need check whether already have one here.